module/entities/helper.js

```diff
@@ -53,15 +54,20 @@ module.exports = class EntitiesHelper {
                 let entityData = await database.models.entities.create(
                     entityDocuments
                 );
+                
+                let entityId = [];
```

Comment:
@deepakr123 - Rename this to entities variable

Reply:
resolved